API.Microsoft.Extensions.Hosting.Testing/WithDocs/Microsoft.Extensions.Hosting/FakeHostingExtensions.cs

```diff
@@ -0,0 +1,83 @@
+
+    /// <summary>
+    /// Gets the object that collects log records sent to the fake logger.
+    /// </summary>
+    /// <param name="host">An <see cref="T:Microsoft.Extensions.Hosting.IHost" /> instance.</param>
+    /// <exception cref="T:System.InvalidOperationException">No collector exists in the provider.</exception>
+    /// <returns>The collector that tracks records logged to fake loggers.</returns>
+    public static FakeLogCollector GetFakeLogCollector(this IHost host);
+
+    /// <summary>
+    /// Gets the object reporting all redactions performed.
+    /// </summary>
+    /// <param name="host">An <see cref="T:Microsoft.Extensions.Hosting.IHost" /> instance.</param>
+    /// <exception cref="T:System.InvalidOperationException">No collector exists in the provider.</exception>
+    /// <returns>The collector that tracks redactions performed on log messages.</returns>
+    public static FakeRedactionCollector GetFakeRedactionCollector(this IHost host);
+
+    /// <summary>
+    /// Adds an action invoked on each log message.
+    /// </summary>
+    /// <param name="builder">An <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</param>
+    /// <param name="callback">The action to invoke on each log message.</param>
+    /// <returns>The <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</returns>
+    public static IHostBuilder AddFakeLoggingOutputSink(this IHostBuilder builder, Action<string> callback);
+
+    /// <summary>
+    /// Exposes <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> for changes via a delegate.
+    /// </summary>
+    /// <param name="builder">An <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</param>
+    /// <param name="configure">Configures the <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</param>
+    /// <returns>The <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</returns>
+    /// <remarks>Designed to ease host configuration in unit tests by defining common configuration methods.</remarks>
+    public static IHostBuilder Configure(this IHostBuilder builder, Action<IHostBuilder> configure);
+
+    public static IHostBuilder ConfigureHostConfiguration(this IHostBuilder builder, params (string key, string value)[] configurations);
+
+    /// <summary>
+    /// Adds a configuration value.
+    /// </summary>
+    /// <param name="builder">An <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</param>
+    /// <param name="key">The configuration key.</param>
+    /// <param name="value">The configuration value.</param>
+    /// <returns>The <see cref="T:Microsoft.Extensions.Hosting.IHostBuilder" /> instance.</returns>
+    public static IHostBuilder ConfigureHostConfiguration(this IHostBuilder builder, string key, string value);
+
+    public static IHostBuilder ConfigureAppConfiguration(this IHostBuilder builder, params (string key, string value)[] configurations);
```

Comment:
No documentation for this method?

Reply:
API review tool bug, the comments are there in the code.